modules/scripts/genindex.py

```diff
@@ -104,27 +96,25 @@ def add_javadoc(index):
      "* `javadoc <./javadoc>`_"]
     return index + javadoc
 
+def add_subprojects(subprojects, index):
+    subprojects_toc_tree = ["Sub Projects", "\n", "================="]
+    subproject = subprojects.split(",")
+    for name in subproject:
+        subprojects_toc_tree.append("\n")
+        subprojects_toc_tree.append("* `{0} <./{0}>`_".format(name))
+    return index + subprojects_toc_tree
+
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('root', help='Path to the root of the Repository')
     parser.add_argument('-s', '--subprojects', default='',
-                        help='Comma seperated subprojects')
+                        help='Comma seperated subprojects name')
```

Comment:
Yeah I understand that. What I'm saying is that the `s` should be with `name` instead of `subproject`.